test/testutils.ts

```diff
@@ -0,0 +1,71 @@
+import type { Session, Transport, UserFeedback } from '@sentry/types';
```

Comment:
Are any of these utils used in the capacitor tests?

Reply:
I think during the development of the migration, this file was required but it is no longer required for the tests so I am removing it, nice catch!